packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -374,9 +374,7 @@ function CheckoutStepGroupWrapper( {
 	// Change the step if the url changes
 	useChangeStepNumberForUrl( store.actions.setActiveStepNumber );
 
-	// Note: the composite-checkout class name is also used by FullStory to avoid recording
-	// WordPress.com checkout session activity. If this class name is changed or removed, we
-	// will also need to adjust this FullStory configuration.
```

Comment:
@Automattic/shilling hi! Should we add HotJar.com ignore props here, or is the checkout page already being ignored by HotJar?

See https://help.hotjar.com/hc/en-us/articles/115012439167

Reply:
Good question! 🤷 I don't know. We didn't implement any of this. See https://github.com/Automattic/wp-calypso/pull/53261 cc @donlair

Reply:
Oh, I see; you're asking about something related but not directly affected by this. I don't know what HotJar is, tbh, so I'm not sure. Is there a way we can find out who uses that and if they care?

Reply:
HotJar.com is a tool used by designers across Jetpack & WP.com (at least) to record sessions, but we should blur out sensitive information. HotJar does pretty good job by default, but I'd rather blur the entire checkout screen.